src/EditorFeatures/Core.Wpf/Preview/PreviewFactoryService.cs

```diff
@@ -396,13 +396,13 @@ public Task<object> CreateChangedDocumentPreviewViewAsync(Document oldDocument,
                 var conflictNodes = newRoot.GetAnnotatedNodesAndTokens(ConflictAnnotation.Kind);
                 var conflictSpans = conflictNodes.Select(n => n.Span.ToSpan()).ToList();
                 var conflictDescriptions = conflictNodes.SelectMany(n => n.GetAnnotations(ConflictAnnotation.Kind))
-                                                        .Select(a => ConflictAnnotation.GetDescription(a))
+                                                        .Select(a => $"❌ {ConflictAnnotation.GetDescription(a)}")
```

Comment:
Do we have any tests on this to make sure we don't accidentally mangle our file encoding and instead spit mojibake to the ueser?

Reply:
We have utf-8-bom and code reviews to prevent that from occurring. 😄 Tests aren't especially helpful here because anything that could break the source version could break the test in the same way.

Reply:
Cute!